node/Cargo.toml

```diff
@@ -8,14 +8,10 @@ categories = ["cryptography", "cryptography::cryptocurrencies"]
 keywords = ["dusk", "cryptocurrency", "blockchain", "node"]
 license = "MPL-2.0"
 
-[[bin]]
-name = "rusk-node"
-path = "src/bin/main.rs"
```

Comment:
As we are moving `the binary` from node to rusk package, could we also move `testbed.sh` or at least file an issue to address this?

Reply:
For the record, I'm actually now updating `testbed.sh`. The comment will be addressed with the next PR.